processor/k8sprocessor/kube/kube.go

```diff
@@ -70,7 +71,7 @@ type Pod struct {
 }
 
 type deleteRequest struct {
-	ip   string
+	id   string
 	name string
```

Comment:
Is this pod name? If yes can we name it `podName`?

Reply:
No, this is identifier (ip or uid)